resources/mongo_scripts/b_create_indexes.js

```diff
@@ -397,7 +397,7 @@ do {
       sleep(retryMilliseconds)
     }
   } catch (err) {
-    console.log('Error while setting up TTL indexes in collections')
+    console.log('Error while setting up TTL indexs in collections')
```

Comment:
```suggestion
    console.log('Error while setting up TTL indices in collections')
```
nit: the [plural of index is either indexes or indices](https://www.merriam-webster.com/dictionary/index)